databricks/koalas/frame.py

```diff
@@ -1436,6 +1436,103 @@ def extract_kv_from_spark_row(row):
             s = pd.Series(v, index=columns, name=k)
             yield k, s
 
+    def itertuples(self, index: bool = True, name: Optional[str] = "Pandas") -> Iterator:
```

Comment:
For the parameter `name`, shall we set it to `Koalas` by default?

The `name` is the name of the numedtuple.

Reply:
+1

Reply:
+1, too